se_shared/test/path_planning/candidateview_unittest.cpp

```diff
@@ -155,10 +158,38 @@ TEST_F(CandidateViewTest, EntropyCalculation_occ) {
 
   //THEN
 }
-TEST_F(CandidateViewTest, EntropyCalculation) {
+TEST_F(CandidateViewTest, getSphereAroundPoint ) {
   //GIVEN:
-
+  // Allocated block: {56, 8, 248};
+  mapvec3i block_voxel_map;
+  const Eigen::Vector3i center = {60, 12, 252};
+  const float radius = 0.2f;// [m] 2 voxels
   //WHEN
+  /* Update bottom left corner as occupied */
+  se::VoxelBlock<testT> *block = oct_.fetch(56, 8, 248);
+  const Eigen::Vector3i blockCoord = block->coordinates();
+  int x, y, z, blockSide;
+  testT low;
+  low.x = 2.f;
+  testT high;
+  high.x = 10.f;
+  blockSide = (int) se::VoxelBlock<testT>::side;
+  int xlast = blockCoord(0) + blockSide;
+  int ylast = blockCoord(1) + blockSide;
+  int zlast = blockCoord(2) + blockSide;
+  for (z = blockCoord(2); z < zlast; ++z) {
+    for (y = blockCoord(1); y < ylast; ++y) {
+      for (x = blockCoord(0); x < xlast; ++x) {
+        if ((blockCoord(0)) < x && x < (xlast - 4) && (blockCoord(1)) < y && y < (ylast - 4)
+            && (blockCoord(2)) < z && z < (zlast - 4)) {
+          block->data(Eigen::Vector3i(x, y, z), low);
+        } else {
+          block->data(Eigen::Vector3i(x, y, z), high);
+        }
+      }
+    }
+  }
+  se::exploration::getSphereAroundPoint(center, radius, oct_, block_voxel_map);
 
   //THEN
```

Comment:
Looks like nothing is tested :stuck_out_tongue: Add a TODO comment so we can find it later